### ISSUE TITLE
Cleric does not seem to go to bed

### ISSUE THREAD

**Issue by LinaBoeckwurm** [TangoTek/TekTopia-Community#212]
I was wondering why my cleric was so sad that he slouched and started to look into it. Turns out he stays up all day to bless people, and up all night to heal the guard. He does not seem to go to bed.

I'm not sure what caused this behavior; I've seen the cleric sleeping before. I've upgraded from the .11 to the .18 version, and I think this may have made the difference in behaviour, but it could be something else as well.

Additional information: I've tried restarting the world to fix this issue.

**Comment by JohnyDL** [comment]
Don't know I've seen mine sleeping in 1.0.18 but they should be a little more ... I don't know have a better shift schedule? if there needs to be one awake at all times then they should take it in turns when they sleep with a little overlap. They unfortunately have both a day job and a night one.

**Comment by LinaBoeckwurm** [comment]
In that case the problem is probably that I only have one. 
Still, I don't think that that would mean he should work constantly and drain all his happiness.

**Comment by JohnyDL** [comment]
I agree, it's a bug. don't know how to solve it, you offline or online experiencing it?

**Comment by LinaBoeckwurm** [comment]
Singleplayer world?

**Comment by JohnyDL** [comment]
hmm how many do you have?

**Comment by Rob2209** [comment]
I got the same Problem on Server 1 **(At 1560 70 5590)**. I got one Cleric who always wanders around the Village and doesn't sleep at all. (Well, he died...)
